### PR TITLE
Bump HerdDB from 0.15.2 to 0.22.0

### PR DESCRIPTION
**Pull request by pv3nturi**

### CODE REVIEW

**Review by eolivelli — CHANGES_REQUESTED** [review]
we are close to the release of HerdDB 0.22.0
I suggest you to wait

File: carapace-server/pom.xml [hunk 1]
```diff
@@ -272,6 +272,12 @@
             <artifactId>route53</artifactId>
             <version>2.13.13</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.bookkeeper.stats</groupId>
+            <artifactId>prometheus-metrics-provider</artifactId>
+            <version>4.12.0</version>
```
Comment (eolivelli) [inline carapace-server/pom.xml:278]: btw this is not the version of BK that is in HerdDB 0.21.0

**Review by eolivelli — CHANGES_REQUESTED** [review]

**Review by aluccaroni — APPROVED** [review]

**Review by aluccaroni — CHANGES_REQUESTED** [review]
@pv3nturi can you ensure that the version of BK is the same officially supported in herddb 0.22?

**Comment by pv3nturi** [comment]
> @pv3nturi can you ensure that the version of BK is the same officially supported in herddb 0.22?
I'll fix it in next commit

**Pull request closed without merging**